src/main/scala/io/qbeast/spark/delta/OTreeIndex.scala

```diff
@@ -81,7 +85,18 @@ case class OTreeIndex(index: TahoeLogFileIndex) extends FileIndex {
     }.toArray
     val stagingStats = stagingFiles
     val fileStats = qbeastFileStats ++ stagingStats
-
+    val execId = spark.sparkContext.getLocalProperty(SQLExecution.EXECUTION_ID_KEY)
+    partitionFilters.foreach(f => {
+      logInfo(s"OTreeIndex partition filter (exec id ${execId}): ${f.toString}")
+    })
+    dataFilters.foreach(f => {
+      logInfo(s"OTreeIndex data filter (exec id ${execId}): ${f.toString}")
+    })
+    val allFilesCount = snapshot.allFiles.count
+    val nFiltered = allFilesCount - fileStats.length
+    val filteredPct = (((nFiltered * 1.0) / allFilesCount) * 100.0).round
+    val filteredMsg = s"${nFiltered} of ${allFilesCount} (${filteredPct}%)"
+    logInfo(s"OTreeIndex filtered files (exec id ${execId}): ${filteredMsg}")
```

Comment:
Instead of `OTreeIndex` could we print Qbeast?

Reply:
OK, done.